CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Unreleased
 
-- **[FIX]** Fix text alignment and image covering for  `MediaContentSection`
 - [...]
 
+# v20.1.2 (12/02/2020)
+
+- **[FIX]** Fix text alignment and image covering for  `MediaContentSection`
```

Comment:
nit: double space here "for  `MediaContentSection`"

Reply:
Done